42_advanced_configuration_management/30_server_policy_generation.txt

```diff
@@ -5,11 +5,30 @@ policy needed by a node, it is necessary to regenerate the modified
 promises for every impacted node. By default this process is launched after each
 change.
 
+The process of policy generation:
+
+* Use configured policies and information about the nodes to generate
+  the files defining the policy that reflects the desired state
+* Compute and store expected reports that will be produced when executing these policies
+* Check the validity of the gfenerated policies
```

Comment:
*gf*enerated

Perhaps the sentence should be: execute post-generation hooks, among which check... But not sure because you are talking about hooks just afterward.